### ISSUE TITLE
interface IX123abc throw an error

### ISSUE THREAD

**Issue by thomas3577** [palantir/tslint#4624]
### Bug Report

- TSLint V5.15.0:
- TypeScript V3.2.4:
- Running TSLint via: CLI, VSCode

#### TypeScript code being linted

```ts
interface II18nService {
}

class I18nService implements II18nService {
}
```

with `tslint.json` configuration:

```json
...
"interface-name": [
  true,
  "always-prefix"
],
...
```

#### Actual behavior
ERROR: .../i18n.service.ts: - interface name must start with a capitalized I

#### Expected behavior
No Error

**Examples that work:**
- IAnything
- IIanything
- INanything

Examples that don't work:
- II18anything
- IN18anything

It seems the numbers in the interface-name are a problem.

And by the way: A Class with the name I18nService works fine.

**Comment by VincentLanglet** [comment]
I did this https://github.com/palantir/tslint/pull/4626

**Comment by chulanovskyi** [comment]
The problem still exists. 

![Screenshot from 2019-04-04 17-55-04](https://user-images.githubusercontent.com/13966263/55565972-7cb0f100-5703-11e9-97ac-004aa16dc609.png)

I suppose that tslint expects that the name of the interface should start from two (?) capital letters? Is this documented somewhere?

**Comment by VincentLanglet** [comment]
@chulanovskyi The PR is not already merge. 

But you don't have the same issue. The doc can be updated this way
> Requires interface names to begin with a capital ‘I’ prefix

tslint expect `IInjectedToggleStateProps` in your case
